addons/mail/static/src/owl/components/composer_text_input/composer_text_input.scss

```
@@ -15,17 +15,40 @@
     overflow: auto
 }
 
-.o_ComposerTextInput_mentionMenuItem {
-    white-space: nowrap;
-    overflow: hidden;
+.o_ComposerTextInput_mention_dropdown .o_mention_proposition {
 
-    &.o-mobile {
-        padding: 5px;
+    display: flex;
+    width: 100%;
```

for size with %, there is the $sizes variable.

```suggestion
    width: map-get($sizes, 100);
```

---

Done